test/nuget/Directory.Build.props

```diff
@@ -14,6 +14,12 @@
         <!-- Build overrides the roots. -->
         <GenerateProjectSpecificOutputFolder>false</GenerateProjectSpecificOutputFolder>
         <OutDirWasSpecified>true</OutDirWasSpecified>
+        
+        <!-- Set default for SolutionDir -->
+        <SolutionDir Condition="'$(SolutionDir)' == ''">$(MSBuildThisFileDirectory)</SolutionDir>
+        <!-- Ensure any SolutionDir has a trailing slash, so it can be concatenated -->
```

Comment:
note, multiple slashes are ok in a file path, the file system treats them as a single slash.